### PR TITLE
GPS IOC update

### PR DESCRIPTION
**Pull request by Pop0097**
Updated the GPS IOC file so the boardfiles reflect required settings.

### CODE REVIEW

File: Autopilot/boardfiles/Autopilot_ZeropilotRev1.ioc [hunk 1]
```diff
@@ -38,27 +61,28 @@ KeepUserPlacement=true
...
-Mcu.IP9=RCC
-Mcu.IPNb=22
+Mcu.IP4=DMA
+Mcu.IP5=FREERTOS
+Mcu.IP6=I2C1
+Mcu.IP7=I2C2
+Mcu.IP8=I2C4
+Mcu.IP9=NVIC
+Mcu.IPNb=23
 Mcu.Name=STM32F765Z(G-I)Tx
```
Comment (Pop0097) [inline Autopilot/boardfiles/Autopilot_ZeropilotRev1.ioc:88]: Something to be concerned about?

File: Autopilot/boardfiles/Inc/FreeRTOSConfig.h [hunk 2]
```diff
@@ -128,7 +128,7 @@ to exclude the API function. */
 #define INCLUDE_vTaskDelete                  1
 #define INCLUDE_vTaskCleanUpResources        0
 #define INCLUDE_vTaskSuspend                 1
-#define INCLUDE_vTaskDelayUntil              1
+#define INCLUDE_vTaskDelayUntil              0
```
Comment (Pop0097) [inline Autopilot/boardfiles/Inc/FreeRTOSConfig.h]: reject, or keep?

**Review by Fberbari — APPROVED** [review]
Really hard to review. But this should be merged asap. I'll address any problems in final integration.

**Comment by Fberbari** [comment]
All I can say is I've opened the IOc and it seems fine. Just ditch any changes in the freertos.c file when resolving merge conflicts and we should be good

**Pull request merged**